### PR TITLE
Release 0.2.4: Changelog & Version

### PR DESCRIPTION
**Pull request by ax3l**
Write the change log for release 0.2.4 and bump the version.

### To Do

- [x] rebase after #1903 & #1906
- [x] incorporate in change log: *Particle manipulators: Fix Position Offset* #1852
- [x] ~~incorporate fix for moving window restart with `slice_point = 0.` #1913~~
- [x] finalize release date

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,35 @@
 Change Log / Release Log for PIConGPU
 ================================================================
 
+0.2.4
+-----
+**Date:** 2017-02-..
```
Comment (PrometheusPi) [inline CHANGELOG.md]: REMINDER:
add release date

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,35 @@
...
+
+Charge of bound electrons, openPMD Axis Range
```
Comment (PrometheusPi) [inline CHANGELOG.md]: OPTIONAL:
lower case axis range

Comment (ax3l) [inline CHANGELOG.md]: this line is the title of the release will proper caps of all words but names and prepositions

Comment (PrometheusPi) [inline CHANGELOG.md]: okay - but wouldn't then the words `bound electrons` be capitalized ?

Comment (ax3l) [inline CHANGELOG.md]: exactly, will also add the particle position offset to the title

Comment (PrometheusPi) [inline CHANGELOG.md]: okay - great

Comment (ax3l) [inline CHANGELOG.md]: will push the updates after rebase :)

**Comment by ax3l** [comment]
@psychocoderHPC @PrometheusPi if someone helps me comparing a `master` vs. a `release-0.2.4` khi e+/e- heating (and visual test, because we changed particle offset maniplators) and testing the LWFA example still runs I can do release today if you like.

**Comment by PrometheusPi** [comment]
Please postpone merging this pull request until issue #1913 is fixed thus the new release can fix this bug.

**Comment by PrometheusPi** [comment]
The mentioned issue is fine and causes no problems - the changelog can be merged. 
For details see #1913.

**Pull request merged**